proposals/PluginsAudit2015/audit.md

```diff
@@ -0,0 +1,223 @@
+# Cordova Plugins
```

Comment:
Would also love to have https://github.com/MobileChromeApps/cordova-plugin-background-app be considered for folding into platforms. Many common patterns on Android don't work well without it (e.g. push notifications)